en/engine/animation/skeletal-animation.md

```diff
@@ -0,0 +1,73 @@
+
+## Skinning algorithm
+
+We have built-in two common standard skinning algorithms, which have similar performance and only affect the final performance:
+
+1. LBS (Linear Mixed Skinning): bone information is stored in the form of a 3x4 matrix, and the matrix is ​​interpolated directly to achieve skinning, and there are typical known problems such as volume loss.
+2. DQS (Dual Quaternion Skinning): The bone information is interpolated in the form of double quaternion, which is more accurate and natural for the skeleton animation without scaling transformation, but for performance reasons, there is an approximate simplified process for all scaling animations.
+
+The engine uses LBS by default. You can switch the skinning algorithm by modifying the `updateJointData` function reference of the engine skeletal-animation-utils.ts and the header file reference in cc-skinning.chunk.
+
+It is recommended that projects with higher pursuit of skin animation quality can try to enable DQS, but since there is no `fma` instruction before GLSL 400, operations such as `cross` cannot bypass floating-point cancellation on some GPUs, and the error is relatively high. Large, may introduce some visible defects.
+
+## Hanging point system
+
+If you need to hang some external nodes to the specified bone joints, you need to use the Socket system of the skeleton animation component:
+  * Create a new child node under the bone animation component to be docked (the parent directly should be the node where the animation component is located).
+  * Add an array element to the sockets property of the skeleton animation component, select the path of the bone to be connected from the drop-down list, and specify the target as the child node just created.
+  * This child node becomes the target hanging point, you can put any external node under this child node, it will follow the specified bone transformation.
+
+The hanging point model in the `FBX` or `glTF` resource will automatically connect to the hanging point system without any manual operation.
+
+## About News Instancing
+
+Based on the frame design of the pre-baking system, the instancing of the skin model has also become a function within reach, but to ensure correctness, you need to collect some relatively low-level information.
+
+The fundamental problem here is that the bone maps used by each model in the same drawcall must be the same. If they are not the same, the display effect will be completely messy.
+
+So how to assign animation data to each bone map becomes a user-defined information, corresponding to the [skeletal map layout panel] set in the editor project (../../editor/project/joints- texture-layout.md) to configure.
```

Comment:
So the way to distribute all the animation data used at runtime to each joint texture atlases becomes a project-specific information, thus needs developer's input. See the [joint texture layout panel](../../editor/project/joints- texture-layout.md) for more details on how to configure this.